index.js

```diff
@@ -15,6 +15,30 @@ module.exports = {
   parser: 'babel-eslint',
   extends: 'airbnb',
   rules: {
-    'valid-jsdoc': ['error']
+    'valid-jsdoc': ['error'],
+
+    // We've decided we don't want this for now as it makes writing JSX more painful.
+    // See https://github.com/airbnb/javascript/issues/1584#issuecomment-335667272
+    "function-paren-newline": 0,
+
+    // False positive on custom propTypes + isRequired. https://github.com/yannickcr/eslint-plugin-react/issues/1389
```

Comment:
I suggest "Disabled whilst false positives still exist with custom propTypes + isRequired.".

Also can we put the "See https://github.com/yannickcr/eslint-plugin-react/issues/1389" on a newline like the previous one.